Documentation/connectors-configuration.md

```diff
@@ -167,6 +167,10 @@ In addition to `id` and `type`, the `ldap` connector takes the following additio
 
 * skipCertVerification: a `boolean`. Skip server certificate chain verification.
 
+* initConn: a `integer`. Number of LDAP Connections to establish on startup. Default: `5`
```

Comment:
It probably will be good enough, yes. LDAP servers are generally really good at connection initialization and teardown.